tests/bluetooth/controller/common/defaults_cmake.txt

```diff
@@ -0,0 +1,37 @@
+#
+# Common include directories and source files for bluetooth unit tests
+#
+
+include_directories(
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/include
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/common/include
+        $ENV{ZEPHYR_BASE}/include/bluetooth
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/util
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/include
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw
+)
+
+#
+# we test functionality of ull_conn, which is included directly in the test file
+# so we do not list it here
+#
```

Comment:
Does this comment actually make sense?

Reply:
But what does ull_conn.c have to do with this?

Reply:
but yeah, removed comment